tls/s2n_server_extensions.c

```diff
@@ -63,7 +63,7 @@ int s2n_server_extensions_send_size(struct s2n_connection *conn)
     total_size += s2n_server_extensions_alpn_send_size(conn);
 
     if (s2n_server_can_send_secure_renegotiation(conn)) {
```

Comment:
s2n_server_can_send_secure_renegotiation(conn) should be moved into s2n_server_renegotiation_info_ext_size(), similarly for s2n_server_can_send_nst() and s2n_server_can_send_secure_renegotiation() into their respective extensions

Reply:
good catch! thank you